integration-cli/environment/clean.go

```diff
@@ -53,7 +53,15 @@ func getPausedContainers(t testingT, dockerBinary string) []string {
 func deleteAllContainers(t testingT, dockerBinary string) {
 	containers := getAllContainers(t, dockerBinary)
 	if len(containers) > 0 {
-		icmd.RunCommand(dockerBinary, append([]string{"rm", "-fv"}, containers...)...).Assert(t, icmd.Success)
+		result := icmd.RunCommand(dockerBinary, append([]string{"rm", "-fv"}, containers...)...)
+		if result.Error != nil {
+			// If the error is "No such container: ..." this means the container doesn't exists anymore,
+			// we can safely ignore that one.
+			if strings.Contains(result.Stderr(), "No such container") {
+				return
```

Comment:
Some questions. If there are multiple errors in icmd.RunCommand, won't this prematurely exit if it encounters No such container before the rest? 

Also the z CI is failing because of " removal of container CID is already in progress", should that also be handled here?

I made a patch before I saw this 😭  😭  that handles that if you think it needs doing here, that you are free to use. https://github.com/docker/docker/compare/master...tophj-ibm:fix-panic-in-teardown-test